libvips/iofuncs/util.c

```diff
@@ -1608,15 +1618,21 @@ vips__temp_dir(void)
 
 		if (!done) {
 			if (!GetTempPath(256, buf))
-				strcpy(buf, "C:\\temp");
+				strcpy(buf, "C:/temp");
 		}
 		tmpd = buf;
 #else  /*!G_OS_WIN32*/
 		tmpd = "/tmp";
 #endif /*!G_OS_WIN32*/
 	}
 
-	return tmpd;
+	/* g_spawn_*() on windows seems to hate backslash in filenames, so we use
+	 * '/' everywhere.
+	 */
+	char *result = g_strdup(tmpd);
+	vips__swap_chars(result, '\\', '/');
+
+	return result;
```

Comment:
Perhaps we should only do this on Windows?
```suggestion
#ifdef G_OS_WIN32
	/* g_spawn_*() on windows seems to hate backslash in filenames, so we use
	 * '/' everywhere.
	 */
	return g_strdelimit(g_strdup(tmpd), "\\", '/');
#else
	return g_strdup(tmpd);
#endif
```
(also changed `vips__swap_chars()` to `g_strdelimit()`)